boule/ellipsoid.py

```diff
@@ -146,6 +146,32 @@ def gravity_pole(self):
         )
         return result
 
+    def _prime_vertical_radius(self, sinlat):
+        r"""
+        Calculate the prime vertical radius for a given latitude
+
+        The prime vertical radius is compute following:
+
+        .. math::
+
+            N(\phi) = \frac{a}{\sqrt{1 - e^2 \sin^2(\phi)}}
+
+        Where $a$ is the semimajor axis and $e$ the first eccentricity.
+
```

Comment:
```suggestion

        This function receives the sine of the latitude as input to avoid repeated 
        computations of trigonometric functions.
```